Sources/AngouriMath/Convenience/MathS.cs

```diff
@@ -356,103 +356,104 @@ public static class Hyperbolic
         public static class Series
         {
             /// <summary>
-            /// Finds the symbolic expression of terms of the Taylor expansion of the given function,
+            /// Finds the symbolic expression of terms of the Maclaurin expansion of the given function,
             /// https://en.wikipedia.org/wiki/Taylor_series
-            /// 
-            /// Do NOT call ToArray() or anything like this on the result of this method. It is 
-            /// infinite iterator. To get a finite result as in a sum of finite number of terms,
-            /// call TaylorExpansion
             /// </summary>
             /// <param name="expr">
-            /// The function, to find Taylor expansion of
+            /// The function to find the Taylor expansion of
             /// </param>
-            /// <param name="exprVariable">
-            /// The variable of this function
-            /// (e. g. if you have expr = Sin("t"), then you may want to use "t" for this argument)
-            /// </param>
-            /// <param name="polyVariable">
-            /// The variable over which you want to see the series
-            /// (e. g. if you want to see the result as f(...) + f(...) * x / ... + f(...) * x^2 / ...,
-            /// then you may want to use "x" for this argument)
+            /// <param name="degree">
+            /// The degree of the resulting taylor polynomial (and the variable in the resulting series)
             /// </param>
-            /// <param name="point">
-            /// The point to differentiate the function at. If it equals 0, you will get
-            /// Maclaurin series.
+            /// <param name="exprVariables">
+            /// The variable/s to take the series over (and the variable the series will be over)
+            /// (e. g. if you have expr = Sin("t"), then you may want to use "t" for this argument)
             /// </param>
             /// <returns>
-            /// An infinite iterator over the terms of Taylor series of the given expression.
+            /// An expression in the polynomial form over the expression variables given in <paramref name="exprVariables"/>
             /// </returns>
-            public static IEnumerable<Entity> TaylorExpansionTerms(Entity expr, Variable exprVariable, Variable polyVariable, Entity point)
-                => Functions.Series.TaylorExpansionTerms(expr, exprVariable, polyVariable, point);
-
+            public static Entity Maclaurin(Entity expr, int degree, params Variable[] exprVariables)
+            {
+                var exprToPolyVars = new (Variable exprVariable, Variable polyVariable, Entity point)[exprVariables.Length];
+                for (int i = 0; i < exprVariables.Length; i++) exprToPolyVars[i] = (exprVariables[i], exprVariables[i], 0);
+                return Functions.Series.TaylorExpansion(expr, degree, exprToPolyVars);
+            }
```

Comment:
It can be rewritten into one line with `Linq`. Like, `=> exprVariables.Select(v =>  (v, v, 0)).ToArray`, you get the idea